xml/System.Diagnostics.CodeAnalysis/AllowNullAttribute.xml

```diff
@@ -32,7 +32,8 @@
   </Attributes>
   <Docs>
     <summary>Specifies that <see langword="null" /> is allowed as an input even if the corresponding type disallows it.</summary>
-    <remarks>To override a method that has a parameter annotated with this attribute, use the <code>?</code> operator. For example: <code>override ISet&lt;Enum&gt; ReadJson(JsonReader reader, Type objectType, ISet&lt;Enum&gt;? existingValue, bool hasExistingValue, JsonSerializer serializer)</code>.</remarks>
+    <remarks>To override a method that has a parameter annotated with this attribute, use the <code>?</code> operator. For example: <code>override ISet&lt;Enum&gt; ReadJson(JsonReader reader, Type objectType, ISet&lt;Enum&gt;? existingValue, bool hasExistingValue, JsonSerializer serializer)</code>. For more information, see <see href="/dotnet/csharp/language-reference/attributes/nullable-analysis">Nullable static analysis</see> in the C# guide.</remarks>
+    <related type="Article" href="/dotnet/csharp/language-reference/attributes/nullable-analysis">Nullable static analysis</related>
```

Comment:
@gewarren Does this tag need to be under a different parent element? It's not showing up on the review site.

Reply:
This looks like the correct place. It seems to me you've uncovered a bug: I don't see any See Also sections being rendered for types, e.g. [InvalidToolsetDefinitionException](https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.exceptions.invalidtoolsetdefinitionexception?view=msbuild-16-netcore). However, I do see one being added for a [method overload](https://docs.microsoft.com/en-us/dotnet/api/microsoft.csharp.csharpcodeprovider.-ctor?view=net-5.0#Microsoft_CSharp_CSharpCodeProvider__ctor_System_Collections_Generic_IDictionary_System_String_System_String__), and [here's the source for that](https://github.com/dotnet/dotnet-api-docs/blob/ea91346510c8d4c693de40d07db73490c9ae1e2f/xml/Microsoft.CSharp/CSharpCodeProvider.xml#L162), so it's in the same place. Do you want to log a bug and assign to @mimisasouvanh ?